### DISCUSSION TITLE
Sanity Check for deploying SO to our MSP customer locations.

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#5563] [Unsupported Versions] [answered]
Hello, 

I'm about to start in on deploying SO sensors to our MSP customers and I'm looking for a sanity check on my approach.  I'm planning on deploying sensor servers out to customer sites and connecting them back over to public internet to SO running in at our datacenter.  It is my understanding that communication between the sensor and manager/search servers is encrypted.

At a high level, is this a good approach?  I can't seem to find fault with it and it seems like an obvious answer, but I just want to make sure.

Thank you, ...Rob

**Comment by trocade** [comment, marked as answer]
I would look into VPN between your manager and the sensor servers if possible.
This is the way our solution is setup and it feels a bit more relaxed than pure internet :)

**Reply by Rbartram** [reply]
Thanks for the info.  Are you using something like OpenVPN directly between the sensor and server, or are you having sensors VPN into a firewall?

**Reply by trocade** [reply]
We have a small form factor physical firewall in front of our environment that handles the VPN, it add some cost but it is worth it for us since we can keep the SO servers as simple as possible.

**Reply by Rbartram** [reply]
Gotcha.  Thank you.

**Reply by wildlyunder** [reply]
We use openvpn too

**Reply by Rbartram** [reply]
Thanks for the feedback.  I ended up going with Wireguard on the Manage server and Sensors.
